projects.json

```diff
@@ -2561,7 +2561,15 @@
     "compatibility": [
       {
         "version": "4.0",
-        "commit": "45f5726304f41e6e6ad5d3d836e89940e165f48d"
+        "commit": "d501cb83be55d67779dd2dc3e2dea53fdf78c39d"
+      },
+      {
+        "version": "4.1",
+        "commit": "d501cb83be55d67779dd2dc3e2dea53fdf78c39d"
+      },
+      {
```

Comment:
@keith Thanks for setting this up! You can go ahead and drop the 4.1 version here, as it actually just gets canonicalized to 4.0 anyway. I tested this new hash locally, and it looks good.

```
./runner.py --swiftc build/compat_macos/install/toolchain/usr/bin/swiftc --projects projects.json --include-actions 'action.startswith("Build")' --swift-branch master --include-repos 'path == "mapper"'
PASS: mapper, 4.0, d501cb, Mapper, generic/platform=iOS
PASS: mapper, 4.0, d501cb, Mapper, generic/platform=macOS
PASS: mapper, 4.0, d501cb, Mapper, generic/platform=tvOS
PASS: mapper, 4.0, d501cb, Mapper, generic/platform=watchOS
PASS: mapper, 4.0, d501cb, Swift Package
PASS: mapper, 4.2, d501cb, Mapper, generic/platform=iOS
PASS: mapper, 4.2, d501cb, Mapper, generic/platform=macOS
PASS: mapper, 4.2, d501cb, Mapper, generic/platform=tvOS
PASS: mapper, 4.2, d501cb, Mapper, generic/platform=watchOS
PASS: mapper, 4.2, d501cb, Swift Package
========================================
Action Summary:
     Passed: 10
     Failed: 0
    XFailed: 0
    UPassed: 0
      Total: 10
========================================
Repository Summary:
      Total: 1
========================================
Result: PASS
========================================
```

Reply:
I've removed 4.1 here!